### DISCUSSION TITLE
An error occured in my cmake when building project,Can anyone tell me how to configure to compile successfully

### DISCUSSION THREAD

**Discussion by 1Reek** [obsproject/obs-studio#5159] [Development Questions] [answered]
libdshowcapture submodule not found!  Please fetch submodules.  win-dshow plugin disabled.a
obs-vst submodule not found! Please fetch submodules or set BUILD_VST=OFF

**Comment by ElectronicWar** [comment, marked as answer]
Follow the build instructions accurately, you did not pull the code recursively and therefore important submodules are missing.
